generated-code-samples-for-review/language/language_classify_gcs.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by protoc-gen-go_gapic. DO NOT EDIT.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+
+	language "cloud.google.com/go/language/apiv1"
+	languagepb "google.golang.org/genproto/googleapis/cloud/language/v1"
+)
+
+// [START language_classify_gcs]
+
+// sampleClassifyText: Classifying Content in text file stored in Cloud Storage
+//
+// gcsContentUri: Google Cloud Storage URI where the file content is located.
+// e.g. gs://[Your Bucket]/[Path to File]
+// The text file must include at least 20 words.
+func sampleClassifyText(gcsContentUri string) error {
+	ctx := context.Background()
+	c, err := language.NewClient(ctx)
+	if err != nil {
+		return err
+	}
+
+	// gcsContentUri := "gs://cloud-samples-data/language/classify-entertainment.txt"
+	req := &languagepb.ClassifyTextRequest{
+		Document: &languagepb.Document{
+			Source: &languagepb.Document_GcsContentUri{
+				GcsContentUri: gcsContentUri,
+			},
+			// Available types: PLAIN_TEXT, HTML
+			Type: languagepb.Document_PLAIN_TEXT,
```

Comment:
I'd lean towards a newline to make it clear which comment is associated with which struct field.

Reply:
Good point! should be easy to fix.